frontend/src/components/dialogs/SecretDialogOpenstack.vue

```diff
@@ -209,7 +233,9 @@ export default {
       applicationCredentialName: undefined,
       applicationCredentialSecret: undefined,
       hideApplicationCredentialSecret: true,
-      validationErrors
+      validationErrors,
+      authenticationMethod: AuthenticationMethod.USER,
+      AuthenticationMethod
```

Comment:
I think it is very confusing if we have the same name twice. I think the constants do not simplify the code but make it more complicated.